src/content/docs/tips/tailwind-v4.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Tailwind v4
+description: Quick set up guide for Tailwind v4
+---
+import { Tabs, TabItem, LinkCard } from "@astrojs/starlight/components";
+
+How to use Tailwind's v4 Vite plugin instead of Astro's Tailwind integration.
+
+Be sure to check out the [Tailwind v4 Blogpost](https://tailwindcss.com/blog/tailwindcss-v4-alpha#using-vite) for details on using v4.
+
+## Install
+
+<Tabs>
+    <TabItem label="npm">
+        ```sh
+        npm install --save-dev @tailwindcss/vite
+        ```
+    </TabItem>
+    <TabItem label="pnpm">
+        ```sh
+        pnpm add -D @tailwindcss/vite
+        ```
+    </TabItem>
+    <TabItem label="yarn">
+        ```sh
+        yarn add --dev @tailwindcss/vite
+        ```
+    </TabItem>
+</Tabs>
+
+## Configure
+
+```ts title="astro.config.mjs"
+import { defineConfig } from 'astro/config';
+import tailwindcss from '@tailwindcss/vite';
+
+// https://astro.build/config
+export default defineConfig({
+  vite: {
+    plugins: [tailwindcss()],
+    css: {
+      transformer: 'lightningcss'
+    }
+  }
+});
+```
+
+## CSS File
```

Comment:
I think this section (`CSS File`) is visually no great yet. If the FileTree component support highlights, I would add it to show which files need to be added. Then  I would maybe use the Step component to walk the user through the setup/tasks step-by-step. And give a little sentence what the "imports" do in the respective files, maybe?

Reply:
ah good point this whole thing should be in steps